server/database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -15,10 +15,13 @@ public function up()
     {
         Schema::create('users', function (Blueprint $table) {
             $table->id();
-            $table->string('name');
+            $table->string('first_name');
```

Comment:
make a new migration file if there are changes you want to update. This is bad practice to edit directly in migration file since it might ruin some functions in production.

Reply:
Okay miss i was used sa pass technology na ako g used wherein if mag edit kog directly kay mo generate siya automatically ug migration file for that change. This is noted miss. Thank you.